mlflow/pyfunc/__init__.py

```diff
@@ -192,26 +198,19 @@ def _copy_file_or_tree(src, dst, dst_dir):
 
 def save_model(dst_path, loader_module, data_path=None, code_path=(), conda_env=None,
                model=Model()):
-    """Export model as a generic python-function model.
+    """Export model as a generic Python function model.
 
    Args:
-       dst_path (str): path where the model is gonna be stored.
-       loader_module (str): the module to be used to load the model.
-       data_path (str): path to a file or directory containing model data.
-       code_path (list[str]): list of paths (file or dir)
-           contains code dependencies not present in the environment.
-           every path in the code_path is added to the python path
-           before the model is loaded.
-       conda_env (str): path to the conda environment definition (.yml).
-           This environment will be activated prior to running model code.
+       dst_path (str): Path where the model is stored.
+       loader_module (str): The module to be used to load the model.
+       data_path (str): Path to a file or directory containing model data.
+       code_path (list[str]): List of paths (file or dir) contains code dependencies not present in
+       the environment. Every path in the ``code_path`` is added to the Python path before the model
+       is loaded.
+       conda_env (str): Path to the Conda environment definition. This environment is activated
+       prior to running model code.
```

Comment:
Maybe we should convert this to ` :param dst_path:` style syntax. We use that in other functions and it will render better in the docs.